package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-paginate",
-  "version": "4.1.1",
+  "version": "4.1.2",
```

Comment:
changing `package` version can happen when multiple PR's are combined. This can be updated before pushing a new version to `npm`